---
title: "Cloud Manager Assets 성능 테스트 실패"
description: "Cloud Manager Assets 성능 테스트가 실패한 이유를 알아보십시오."
solution: Experience Manager
product: Experience Manager
applies-to: "Experience Manager"
keywords: "KCS, cloud manager, assets 성능, 오류, Experience Manager"
resolution: Resolution
internal-notes: "https://jira.corp.adobe.com/browse/CQDOC-21090"
bug: false
article-created-by: Zita Rodricks
article-created-date: "5/30/2024 3:03:30 PM"
article-published-by: Zita Rodricks
article-published-date: "5/30/2024 3:09:26 PM"
version-number: 5
article-number: KA-23312
dynamics-url: "https://adobe-ent.crm.dynamics.com/main.aspx?forceUCI=1&pagetype=entityrecord&etn=knowledgearticle&id=f49f18c0-951e-ef11-840a-000d3a372703"
source-git-commit: 7c349b64c537d0994f04df0e073955d8bab2c8cd
workflow-type: tm+mt
source-wordcount: '332'
ht-degree: 0%

---

# Cloud Manager Assets 성능 테스트 실패


알 수 없는 오류로 자산 성능 테스트가 실패했습니다. 이 문제가 발생하는 이유를 알아보십시오.

## 설명 {#description}


### 환경

AEM Managed Services

### 문제/증상 

알 수 없는 오류로 자산 성능 테스트가 실패했습니다.

<b>단계 실패</b>: assetsTest

실패한 단계 assetsTest의 직접 로그를 사용할 수 없습니다. 오류 메시지는 다음과 같습니다.

`{"type":"http://ns.adobe.com/adobecloud/missing-step-logs","status":404,"title":"Step logs not found","errors":[ "Logs not found for programId=809, executionId=3473820, phaseId=6253498, stepId=11687695"] } `

그러나 pipelineExecutionFailureReasons는 `ASSETS_TESTS_AUTHOR_CONNECTION_ERROR` 오류 코드 포함 `002_010_006` 그리고 메시지는 다음과 같습니다.

`Unable to connect to the author instance. `

내부 세부 정보는 을 나타내는 메시지와 함께 HTTP 응답 문제를 제안합니다. *찾을 수 없음*: 일반적으로 테스트 실행 중에 작성자 인스턴스의 끝점에 연결할 수 없음을 의미합니다.

assetsTest 단계 중에 작성자 인스턴스에 대한 연결 오류가 발생하여 테스트가 성공적으로 완료되지 않는 것이 주요 문제입니다.

그러나 Cloud Manager 사용자가 존재하며 작성자에 자산을 수동으로 업로드하는 것은 예상대로 작동합니다.

에셋 성능 테스트가 30분 동안 작동하고 30분 후 테스트가 중단되는 동안 반복적인 방식으로 갑자기 실패합니다.

다음을 참조하십시오 [AEM Assets](https://experienceleague.adobe.com/docs/experience-manager-cloud-manager/content/using/code-quality-testing.html#aem-assets)AEM 6.x용 Cloud Manager 설명서에서 참조할 수 있습니다.


## 해결 방법 {#resolution}


내부 Azure 함수 로그를 확인하는 동안 요청에 404 Not Found 결과가 표시되었습니다. 에서 URL로의 GET *메시지* 필드가 404를 반환했습니다. 아래를 참조하십시오.

이 경우의 실패 원인은 Adobe의 권장 사항에 따라 QueryBuilder가 Dispatcher 수준에서 차단되었기 때문입니다.
그러나 Cloud Manager는 에셋 테스트 중처럼 Cloud Manager 사용자를 위한 Dispatcher에서 QueryBuilder를 허용해야 합니다. 처리 시간이 검색됩니다.




```
{
 "x-ms-workflow-run-id": " ",
 "x-ms-workflow-operation-name": "1500_measureAssetsProcessingTime",
 "message": "Req: GET https://author-URL.adobecqms.net/bin/querybuilder.json?p.hitwriter=selective&p.limit=1000&p.properties=jcr%3Acontent%2Fjcr%3AlastModified+jcr%3Acreated&path=%2Fcontent%2Fdam%2Fcloudmanager&property=jcr%3Acontent%2Fdam%3AassetState&property.depth=1&property.value=processed&type=dam%3AAsset",
 "x-ms-client-tracking-id": "trackingID",
 "trackingContext": {
 "executionId": executionID,
 "tenantId": tenantID,
 "imsOrgId": "IMS_ORG",
 "programId": programID,
 "pipelineId": pipelineID
 }
}
```






<b>Experience League 캠페인 커뮤니티에서 질문하기</b>

이 주제에 대해 답변이 필요하거나 이전 답변이 된 질문 을 읽으려는 경우 다음을 확인하시기 바랍니다. [Experience League 커뮤니티 블로그](https://experienceleaguecommunities.adobe.com/t5/adobe-experience-manager-blogs/introducing-top-kcs-articles-curated-for-your-aem/ba-p/672734#M1180) 이 문서가 포함된 게시물을 올리고, 질문과 의견을 보내며, Experience League 캠페인 커뮤니티에 참여하십시오!


